labrad/types.py

```diff
@@ -1319,8 +1337,8 @@ def __flatten_array__(self, a, endianness):
         else:
             elems = (self.elem.flatten(i, endianness=endianness).bytes
                      for i in a.flat)
-            return dims + ''.join(elems), self
-        return dims + a.tostring(), self
+            return dims + b''.join(elems), self
+        return dims + a.tobytes(), self
```

Comment:
In Python 2.7, won't this throw an exception? Or does the `future` library monkey patch strings?